### ISSUE TITLE
yandex.ru: ads

### ISSUE THREAD

**Issue by EntonyHop** [uBlockOrigin/uAssets#25604]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I am not using uBO Lite along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBOL Lite and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://yandex.ru/video/preview/8732949103562496470`

### Category

ads

### Description

the ad's not gone.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

```yaml
version: uBOL 2024.10.6.1334
mode: complete / optimal
rulesets:
 default
 rus-0
 dpollock-0
 adguard-spyware-url
 annoyances-cookies
 annoyances-social
 annoyances-others
 stevenblack-hosts
 annoyances-overlays
 annoyances-widgets
 block-lan
```

**Comment by dportvine** [comment]
@dimisa-RUAdList

**Comment by stephenhawk8054** [comment]
I think it needs `RU AdList: Counters` which might need some time until it's implemented as a non-default sub-list: https://github.com/uBlockOrigin/uBOL-home/discussions/37#discussioncomment-10158453

**Comment by dimisa-RUAdList** [comment]
@stephenhawk8054 
Current mv3 limits allow additional filters. Do you know if we will see this feature in uBOL?

**Comment by dimisa-RUAdList** [comment]
@stephenhawk8054 
I have not studied this issue, but in Adblock Plus for Google Chrome, a subscription to additional filters is available now.

**Comment by stephenhawk8054** [comment]
@dimisa-RUAdList Yeah, because they don't use declarative approach as uBOL, which also makes the extension having to awaking up its service worker regularly, which sacrifices perfomance and reliability (you can watch videos to see why, especially the reliability): https://github.com/uBlockOrigin/uBOL-home/wiki/Frequently-asked-questions-(FAQ)#is-ubo-lite-a-bad-faith-attempt-at-converting-ubo-to-mv3

**Comment by gorhill** [comment]
What's the official link to the "RU AdList: Counters" list?

**Comment by mapx-** [comment]
https://easylist-downloads.adblockplus.org/cntblock.txt

https://github.com/easylist/ruadlist/wiki/RU-Adlist-links

**Comment by dimisa-RUAdList** [comment]
Most likely, Yandex will start making attempts to bypass those solutions that are implemented in Counters. As a result, these solutions will stop working. It is better to make this filter optional, disabled by default.

**Comment by stephenhawk8054** [comment]
Since the issue is more about adding more lists to uBOL, should we move to `uBOL-home` repo?
